website/server/libs/stringUtils.js

```diff
@@ -4,12 +4,20 @@ export function removePunctuationFromString (str) {
 
 // NOTE: the wordsToMatch aren't escaped in order to support regular expressions,
 // so this method should not be used if wordsToMatch contains unsanitized user input
+
 export function getMatchesByWordArray (str, wordsToMatch) {
+  // remove accented characters from the string, which would trip up the regEx
+  // later on, by using the built-in Unicode normalisation methods
+  // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/normalize
+  // https://www.unicode.org/reports/tr15/#Canon_Compat_Equivalence
+
+  const normalizedStr = str.normalize('NFD').replace(/[\u0300-\u036f]/g, '');
```

Comment:
could you point me to where the list of accented unicode characters can be found?

Reply:
Sure. Please see the Unicode table [here](https://unicode-table.com/en/#combining-diacritical-marks). You notice that the section I have linked to contains all the diacritical marks. 

My understanding is that our accented characters like é were being parsed as the diacritical mark then the standard letter (e.g. `[accent] e`), which broke the regex. I think the normalize function converts `[accent] e => e` under the hood.

The highlight section are the marks we are normalizing with this change: 

![Screen Shot 2020-08-15 at 14 27 11](https://user-images.githubusercontent.com/11768967/90312370-3d540080-df04-11ea-9de4-888963945336.png)

Reply:
I think I should probably also add the link to the relevant unicode table section in the code comments, so I'll do that when I'm adding the new test.

Reply:
thanks! adding a link to the table is definitely a good idea